pythonSynapseUtils/annotationsYaml.py

```diff
@@ -0,0 +1,137 @@
+import yaml
+
+
+def checkAgainstDict(annotDict, project, syn){
+	''' Compares annotations in use in a project against those specified by a dictionary and prints the ones that do not match the dictionary.'''
+	yamlEnt = syn.get(annotDict)
```

Comment:
confusing that annotDict is really a Synapse Entity ID. Docs will help, but explicit variable name would be good (`yamlAnnotSynId`)